### DISCUSSION TITLE
Why does the multiple select not behave like a select?

### DISCUSSION THREAD

**Discussion by DrewCCannedy** [twbs/bootstrap#38580] [Q&A]
https://getbootstrap.com/docs/5.0/forms/select/

In the select multiple example, the component does not actually behave at all like a select. It is a selectable list. Is this intended? And if so, why not implement a full multi select?

**Comment by julien-deramond** [comment]
> Is this intended?

This is the default behavior of the browsers (see https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select_multiple or https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/multiple#try_it).

From one of these links:
> When `multiple` is specified, most browsers will show a scrolling list box instead of a single-line dropdown.
